src/chart/cartesianChart.js

```diff
@@ -19,32 +19,53 @@ export default function(xScale, yScale) {
         },
         yLabel = '',
         xLabel = '',
+        xBaseline = noop,
```

Comment:
Are you using `noop` rather than null because it's being run through `d3.functor`? If so, it's probably worth adding a comment explaining that (it took me a little while to work that out).